bigbluebutton-html5/imports/ui/components/audio/audio-graphql/audio-captions/live/component.tsx

```diff
@@ -35,7 +35,7 @@ const AudioCaptionsLive: React.FC<AudioCaptionsLiveProps> = ({
                       color={user.color}
                       moderator={user.isModerator}
                     >
-                      {user.name.slice(0, 2)}
+                      {user.avatar ? '' : user.name.toLowerCase().slice(0, 2)}
```

Comment:
```suggestion
                      {user.avatar ? '' : user.name.slice(0, 2)}
```

Why to lowercase? Let's keep it matching the userlist version?